apps/io-services-cms-webapp/src/utils/converters/service-lifecycle-converters.ts

```diff
@@ -111,6 +119,19 @@ const toServiceTopic =
         )
       : TE.right(undefined);
 
+//FIXME: remove this workaround for managing the creation and the update of services from pagopa
+//        where the name of the organization needs to be renamed into "IO - L'app dei servizi pubblici"
+const IO_APP_ORGANIZATION_NAME = "IO - L'app dei servizi pubblici";
+const PAGOPA_ORGANIZATION_FISCAL_CODE = "15376371009";
+
+const renameIoAppServicesOrganizationName = (
```

Comment:
Consider to rename this fn into `renamePagoPAServicesOrganizationName` because, imo, the concept is "I rename the org.name for PagoPA services"